PAPER/FIT_Paper_Ex1/Part_1.tex

```diff
@@ -1,52 +1,57 @@
-Since the 1990's, many different nuclear fuel cycle simulation codes have been
+Since the 1990's, many different nuclear fuel cycle simulators have been
 developed by several institutions.
 
-A fuel cycle code aims to models an entire nuclear fleet including main
-facilities, like nuclear reactors, fuel fabrication plants, reprocessing plants,
-cooling pool and waste disposal. Those tools help to identify drivers and
-interactions between parameters in fuel cycle physic. For that, they often
-implement physic models for different key points of the cycle like fuel
-fabrication or burn up depletion calculations. 
+A fuel cycle simulator aims to models an entire nuclear fleet including main
+facilities, such as nuclear reactors, fuel fabrication plants, reprocessing
+plants, cooling pool and/or waste disposal. Those tools help to identify drivers
+and interactions between parameters in fuel cycle. They implement physic models
+for different key points of the cycle such as fuel fabrication or burn up
+depletions, with various level of complexicity. 
 
```

Comment:
A fuel cycle simulator aims to models an entire nuclear fleet including main
facilities, such as nuclear reactors, fuel fabrication plants, reprocessing
plants, cooling pool and/or waste disposal. They implement physic models
for different key points of the cycle such as fuel fabrication or burn up
depletions, with various level of complexicity. Those tools help to identify drivers
and interactions between parameters in fuel cycle.